server/claim/types/sentence/sentence.service.ts

```diff
@@ -123,7 +111,45 @@ export class SentenceService {
                     },
                 },
             },
-        ]);
+        ];
+
+        if (searchText) {
+            pipeline.splice(0, 0, {
```

Comment:
suggestion: The splice logic is a bit confusing to read, I suggest that we just push them in the order that matters. It seems that you're using splice just to add the pipelines in the right order, maybe we should just push them in the right order starting from an empty array, what do you think?

Reply:
I didn't have this ideia, thanks for the suggestion.